website/src/components/canvas/CanvasView.jsx

```diff
@@ -40,13 +45,21 @@ class CanvasView extends React.Component {
    */
   componentDidMount() {
     firebase.database().ref('/test').once('value').then((elemListSnap) => {
-      this.props.dispatch(initPositions(elemListSnap.val()));
+      this.props.dispatch(initElements(elemListSnap.val()));
     });
 
     firebase.database().ref('/test').on('child_changed', (elemSnap) => {
```

Comment:
I agree, not sure how to make this better right now though. Shouldn't have an impact right now though I don't think, just a bit sloppy